### ISSUE TITLE
Version on the Play Store crashes on start (Samsung S4 -- Android 4.4.2)

### ISSUE THREAD

**Issue by bainzo** [guardianproject/notecipher#31]
The version of NoteCipher (0.0.7.1) available on the Play Store crashes on start with the message, `Unfortunately, NoteCipher has stopped.`

Phone is Samsung Galaxy S4 (GT-l9505) running Android 4.4.2

Cheers

**Comment by willydiaz761** [comment]
I also have the problem that I can not enter the app and my android version is 4.4.2 :(

**Comment by willydiaz761** [comment]
this work in Android 4.4.2!!!! https://mega.co.nz/#!8sF20YoL!I6za74gG4BLNGq9Wdiw6kgcNo_MazJV-jLVrmOpxot8

**Comment by ghost** [comment]
Same issue occurs with Nexus 4 running android 4.4.4 (also using ART as the runtime).

**Comment by silvasurfer** [comment]
Same issue on Moto G running NON rooted Android KitKat 4.4.4.
Crashes as soon as its opened. Really want this to work...Uninstalling for now.

**Comment by tomatopipps** [comment]
yep, it doesn't work on non-rooted Samsung Galaxy Ace 4 running Android 4.4.4 either.

**Comment by m8ram** [comment]
Also crashes on my LG G3, Android version 5.0.
No error information just "Unfortunately, NoteCipher has stopped."
Package version: 11
Package version name: 0.0.7.1
SDK version 21
There are at least three other reports of this crash on the LG G3 in the play store.

I used the report function but just in case you don't get that the stack trace starts with

```
java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_ZN7android10MemoryBaseC1ERKNS_2spINS_11MemoryHeapEEEIj"
referenced by "libdatabase_sqlcipher.so"...
at java.lang.Runtime.loadLibrary(Runtime.vava:371)
```
